central/role/service/service_impl.go

```diff
@@ -53,6 +60,8 @@ var (
 		allow.Anonymous(): {
 			"/v1.RoleService/GetResources",
 			"/v1.RoleService/GetMyPermissions",
+			"/v1.RoleService/GetClustersForPermission",
+			"/v1.RoleService/GetNamespacesForClusterAndPermission",
```

Comment:
I think you want them under `user.With():{}` section to require successful authentication.